### ISSUE TITLE
Add support for latest Safari browser on SauceLabs

### ISSUE THREAD

**Issue by alexanderkranga** [serenity-bdd/serenity-core#2512]
Hi,

We have been successfully running our tests using Serenity on SauceLabs in Safari browser for more than 2 years and it works super great!

But, SauceLabs have support for Safari version 14 on `macOS Big Sur` which seems to be missing in the framework. I think that [this file](https://github.com/serenity-bdd/serenity-core/blob/master/serenity-core/src/main/java/net/thucydides/core/webdriver/capabilities/SetAppropriateSaucelabsPlatformVersion.java) is responsible for starting remote sessions on `macOS` in `Safari`.

According to [SauceLabs configurator](https://saucelabs.com/platform/platform-configurator#/) we can use following configuration for latest browser:
```
MutableCapabilities sauceOptions = new MutableCapabilities();

SafariOptions browserOptions = new SafariOptions();
browserOptions.setCapability("platformName", "macOS 11");
browserOptions.setCapability("browserVersion", "14");
browserOptions.setCapability("sauce:options", sauceOptions);
```
or with `latest` version:
```
MutableCapabilities sauceOptions = new MutableCapabilities();

SafariOptions browserOptions = new SafariOptions();
browserOptions.setCapability("platformName", "macOS 11");
browserOptions.setCapability("browserVersion", "latest");
browserOptions.setCapability("sauce:options", sauceOptions);
```

Would really love new Safari version support to be added to Serenity!

Thank you!

**Comment by wakaleo** [comment]
Could you propose a PR for this?

**Comment by wakaleo** [comment]
Hi @alexanderkranga, do you have (or could you provide) a sample project with the saucelabs config that you need? In 2.4.49 there will be no constraints on the properties you put in the `saucelabs` block in the serenity.conf file, but I would like to make sure it works with some real examples.

**Comment by wakaleo** [comment]
Actually this may be limited by the fact that Selenium does not officially support "macOS 11" as a platform (as defined in the `Platform` enum)

**Comment by alexanderkranga** [comment]
Hi @wakaleo , unfortunately I cannot provide a sample project without showing our SauceLabs credentials. I will try to build and publish a local version of serenity and run it against macOS 11

**Comment by wakaleo** [comment]
Presumed fixed in [4d90983](https://github.com/serenity-bdd/serenity-core/commit/4d90983341690eace023c0bfa6b6e50f9403a28e)
